.buildkite/pipeline.yml

```diff
@@ -39,6 +39,7 @@ docker_plugin_sgx_config: &docker_plugin_sgx_config
     - /dev/isgx
   environment:
     - "OASIS_TEE_HARDWARE=intel-sgx"
+    - "OASIS_UNSAFE_ALLOW_DEBUG_ENCLAVES=1"
```

Comment:
why is this alone here? could this be the wrong place to put it?

Reply:
For go, `OASIS_UNSAFE_ALLOW_DEBUG_ENCLAVES` is a runtime parameter. We could use `-ldflags` switch to set a compile-time flag, but this seemed even worse.

Reply:
ah, right. let's unify discussion on Jernej's comment about that